build_config/webpack.dev.conf.js

```diff
@@ -26,6 +26,10 @@ module.exports = {
         test: /\.(ttf|eot|svg|gif)(\?v=[0-9].[0-9].[0-9])?$/,
         loader: "file-loader?name=[name].[ext]"
       }
+    ],
+    // http://andrewhfarmer.com/aws-sdk-with-webpack/
+    noParse: [
```

Comment:
I don't know webpack much either. We could do some Tools training if there are enough of us that are a bit unsure of best practises?